tests/integration/Makefile

```diff
@@ -0,0 +1,48 @@
+SPIN?=spin
+
+# We are testing the sources, not the installed application
+SDL2PROMELA=PYTHONPATH=../../ python3 -m sdl2promela.sdl2promela
+
+RES_DIR=resources
+
+TEST_DIR=out
+
+CREATE_TEST_DIR=mkdir -p ${TEST_DIR}
+DELETE_TEST_DIR=rm -r ${TEST_DIR}
+
+TESTS = \
+	test_bare_outputs
+
+.SILENT : \
+	${TESTS}
+
+.PHONY : \
+	${TESTS} \
+	check
+
+check: ${TESTS}
+
+##############################INTEGRATION TESTS################################
+
+# These tests are created based on the tests supplied with custom Spin version
+# used by N7 Space (see github.com/n7space/Spin). Their purpose is to validate
+# whether the generated Promela code actually works inside Spin.
+# As sdl2promela generates only inlines (representing the SDL logic) which need
+# external data and process definitions (similarly to opengeode generating
+# only C functions, which need type declarations generated by asn1scc and
+# glue code generated by Kazoo), they are wrapped inside "host" Promela
+# files
+
+test_bare_outputs:
+	echo "Test bare outputs"
+	${CREATE_TEST_DIR}
```

Comment:
suggestion - add `-f` to delete and add `delete` before `create`, just in case something crashes during tests and the next run will start from clean output. I'd even consider removing `delete` from the end of the tests to have the opportunity to investigate output after tests

Reply:
renamed to CREATE_EMPTY_TEST_DIR and added rm -r -f to the call